im3components/wrf_tell/wrf_to_tell_part_two.py

```diff
@@ -0,0 +1,149 @@
+# wrf_to_tell_part_two.py
+# Casey D. Burleyson
+# Pacific Northwest National Laboratory
+# 27-Aug 2021
+
+# This script takes the .csv files of average WRF meteorology by county produced by
+# wrf_to_tell_part_one.py and aggregates them into annual hourly time-series of
+# population-weighted meteorology for each balancing authority (BA). This is the
+# second step in a processing chain to go from the WRF output files to the input
+# files needed for TELL. All times are in UTC. Missing values are reported as -9999.
+
+# Import all of the required libraries and packages:
+import glob
+import numpy as np
+import pandas as pd
+import os
+import datetime
+from datetime import datetime as dt
+import re
+
```

Comment:
This needs to be rescoped and potentially added to the `wrf_tell_counties` component.